cpg_workflows/stages/gatk_sv/gatk_sv_common.py

```diff
@@ -214,9 +207,9 @@ def add_gatk_sv_jobs(
         out_path = expected_out_dict[key]
         if isinstance(resource, list):
             for source, dest in zip(resource, out_path):
-                cmds.append(f'gsutil cp "$(cat {source})" "{dest}"')
+                cmds.append(f'gcloud storage cp "$(cat {source})" "{dest}"')
         else:
-            cmds.append(f'gsutil cp "$(cat {resource})" "{out_path}"')
+            cmds.append(f'gcloud storage cp "$(cat {resource})" "{out_path}"')
```

Comment:
incidental change from gsutil to gcloud, long overdue but not specifically relevant to this change